L1Trigger/Phase2L1ParticleFlow/src/newfirmware/puppi/linpuppi_ref.cpp

```diff
@@ -230,6 +233,57 @@ void l1ct::LinPuppiEmulator::linpuppi_chs_ref(const PFRegionEmu &region,
   }
 }
 
+void l1ct::LinPuppiEmulator::linpuppi_chs_ref(const PFRegionEmu &region,
```

Comment:
Don't add a new copy of the functions, just change the existing ones to take a vector of PVObjs.
That should also make the changes easier to review.
I'll review the rest of the changes to these files once this is done

Reply:
updated